src/qt/rpcconsole.cpp

```diff
@@ -1114,11 +1114,11 @@ void RPCConsole::updateDetailWidget()
         peerAddrDetails += "<br />" + tr("via %1").arg(QString::fromStdString(stats->nodeStats.addrLocal));
     ui->peerHeading->setText(peerAddrDetails);
     ui->peerServices->setText(GUIUtil::formatServicesStr(stats->nodeStats.nServices));
-    ui->peerRelayTxes->setText(stats->nodeStats.fRelayTxes ? tr("Yes") : tr("No"));
+    ui->peerRelayTxes->setText(stats->nodeStats.fRelayTxes ? tr_yes : tr_no);
     QString bip152_hb_settings;
-    if (stats->nodeStats.m_bip152_highbandwidth_to) bip152_hb_settings = tr("To");
-    if (stats->nodeStats.m_bip152_highbandwidth_from) bip152_hb_settings += (bip152_hb_settings.isEmpty() ? tr("From") : "/" + tr("From"));
-    if (bip152_hb_settings.isEmpty()) bip152_hb_settings = tr("No");
+    if (stats->nodeStats.m_bip152_highbandwidth_to) bip152_hb_settings = tr_to;
+    if (stats->nodeStats.m_bip152_highbandwidth_from) bip152_hb_settings += (bip152_hb_settings.isEmpty() ? tr_from : "/" + tr_from);
```

Comment:
While here, why not s/`"/"`/`QLatin1String("/")`/ ?

Reply:
Done, and also hoisted repeated translatables "never", "unknown" and "Ban for".